applications/nrf_desktop/Kconfig.ble

```diff
@@ -132,6 +140,12 @@ config BT_ATT_SENT_CB_AFTER_TX
 	  latency of provided HID data. The ATT sent callback is used by the
 	  application to trigger sending subsequent HID report.
 
+config BT_CONN_TX_MAX
+	default BT_ATT_TX_COUNT
+	help
+	  For an nRF Desktop peripheral, the maximum number of pending TX
```

Comment:
why don't we have this alignment in place also for the BT central?

Reply:
I also noticed that `CONFIG_BT_CONN_TX_MAX` is currently marked as `DEPRECATED`. Can we avoid changing configuration in scope of this PR and fix it under a separate ticket: `NCSDK-34284` ? (we should generally stop using the deprecated option)

Reply:
sure